kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/actions/SingleFileConvertAction.java

```diff
@@ -89,6 +103,9 @@ public void setMediaServerUtils(MediaServerUtils mediaServerUtils) {
      */
     public InputStream perform(Work work, Map<String, String> parameter) throws Exception {
 
+        StringBuffer notification = notifier.init();
```

Comment:
I still don't like to keep this buffer here. This should be done by the Notifier and the Notifier should be one instance for a request.
What about using a factory? We could autowire a factory that carries the needed attributes from the context. And here we would create a new instance of notifier using the factory bean like `INotifier notifier = notifierFactory.createNotifier();`
It's not a real factory, I think. Because it's an instance method and not a static one.
What do you think?

Reply:
Could need help with this one. I created an AbstractBeanFactory but have trouble to inject the message within Notifier.